packages/css-library/src/stylesheets/_uswds-theme.scss

```diff
@@ -0,0 +1,28 @@
+@use "uswds-core" with (
+  $theme-show-compile-warnings: false,
+  $theme-show-notifications: false,
+  $theme-font-path: '../node_modules/@uswds/uswds/dist/fonts',
+  $theme-font-type-sans: 'source-sans-pro',
+  $theme-font-sans-custom-stack: '"Source Sans Pro", "Helvetica Neue", Helvetica, Arial, sans',
+  $theme-namespace: (
+    'grid': (
+      namespace: 'vads-grid-',
+      output: true,
+    ),
+    'utility': (
+      namespace: 'u-',
+      output: false,
+    ),
+  ),
+  $theme-utility-breakpoints: (
+    "card": false,
+    "card-lg": false,
+    "mobile": true,
+    "mobile-lg": false,
+    "tablet": true,
+    "tablet-lg": false,
+    "desktop": false,
+    "desktop-lg": true,
+    "widescreen": false
```

Comment:
Consolidating the theme settings to avoid duplication. The `$theme-namespace` and `$theme-utility-breakpoints` settings are used in `utilities.scss` for generating the utilities classes.

Reply:
I like the consolidation here :+1: